conman/redirects/models.py

```diff
@@ -13,9 +14,11 @@ class RouteRedirect(Route):
     This model holds the data required to make that connection.
     """
     handler = handlers.RouteRedirectHandler.path()
-    target = models.ForeignKey('routes.Route', related_name='+')
+    target = models.ForeignKey('routes.Route', related_name='+', on_delete=models.CASCADE)
```

Comment:
This will be [required in Django 2.0](https://docs.djangoproject.com/en/1.10/releases/1.9/#foreignkey-and-onetoonefield-on-delete-argument).